core/tests/browse_tests.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+from django.test import TestCase
+from django.db import connection
+from django.conf import settings
+
+from chronam.core.views.browse import get_page_text
+from chronam.core.utils.utils import _get_tip
+from chronam.core.batch_loader import BatchLoader, Batch
+from chronam.core.models import Title, LanguageText
+
+
+class BrowseTests(TestCase):
+    """
+    Tests related to core/views/browse.py.
+    test_full_text_deleted - 'sanity' check that 'text' column is removed
```

Comment:
Could you move line 16, 17, and 18 to their respective tests, and be more descriptive then `test_getting_text_from_solr_utah and test_getting_text_from_solr_slovenia`?